koala-app/apps/api/src/app/media/entities/media.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { ObjectType, Field, Int, registerEnumType } from '@nestjs/graphql';
+import { isEnum, IsEnum, IsNotEmpty } from 'class-validator';
+import { type } from 'os';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from '../../core/base.entity';
+
+
+export enum MediaType {
+  AUDIO = "audio"
+}
+
+registerEnumType(MediaType, {
+  name : "MediaType"
+});
+
+@ObjectType()
+@Entity()
+export class Media extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  @Field(() => Int, { description: 'ID for Media' })
+  id: number;
+
+  @Column({
+    type: "simple-enum",
+    enum: MediaType
+  })
+  @Field(() => MediaType, { description: 'Media Type' })
+  @IsEnum(MediaType)
+  @IsNotEmpty()
+  type: MediaType;
+
+  @Column()
+  @Field({ description: 'Media Title' })
+  @IsNotEmpty()
+  title: string;
+
+  @Column()
+  @Field({ description: 'Media Composer' })
+  @IsNotEmpty()
+  composer: string;
```

Comment:
I would not make this mandatory with us not empty since this might just be additional information maintained by the owner which might not be needed in all scenarios (especially not to play the audio file) ... same applies to the title field

Reply:
removed